scheduler/src/cook/mesos/task.clj

```diff
@@ -127,10 +127,10 @@
 
 (defn job->task-metadata
   "Takes a job entity, returns task metadata"
-  [db mesos-run-as-user job-ent task-id]
+  [mesos-run-as-user job-ent task-id compute-cluster]
```

Comment:
Nitpick: can we make `compute-cluster` the first argument? Keeping the more 'static' value as the first arguments helps with currying. I feel it also helps with readability in `a->b` named functions when `a` is moved as far to the right as possible in parameter order.

Reply:
Good call, done